cluster/addons/calico-policy-controller/rbac.yaml

```diff
@@ -0,0 +1,87 @@
+kind: ClusterRole
+apiVersion: rbac.authorization.k8s.io/v1beta1
```

Comment:
rbac has updated to v1

Reply:
@xilabao we're planning to port this with k8s 1.7.x, so keeping it `v1beta1` for this PR